src/data.js

```diff
@@ -78,6 +78,25 @@ const pages = [
       'Blogger',
     ],
   },
+  {
+    name: 'Conrad Mugisha',
+    description:
+      'Software Developer. Lover of challenges',
+    url: 'https://github.com/conrad96',
```

Comment:
This needs to be a "uses" URL, I think we're allowing Gist's containing your equipment & software if you don't have a site where you can add the page.